components/canvas_traits/webgl.rs

```diff
@@ -2,8 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use nonzero::NonZeroU32;
 use euclid::Size2D;
+use nonzero::NonZeroU32;
```

Comment:
Please squash this commit ("Make tidy happy") with the rest of the changes.

Reply:
Done.